test/it/negative.spec.js

```diff
@@ -3,12 +3,8 @@ const listSubDirectories = require("../utils/listSubDirectories");
 const writeTypingsHelper = require("../utils/writeTypingsHelper");
 const testParser = require("../utils/testParser");
 
-const {
-  declarations,
-  getWidgetTypeDeclarations,
-  getPageElementsTypeDeclarations,
-  getNpmDependenciesTypesDeclarations
-} = require("../../dist/corvidTypes.umd.js");
+const getDynamicTypings = require("../utils/dynamicTypings");
+const { declarations } = require("../../dist/corvidTypes.umd.js");
```

Comment:
@nemni8 are we using the "umd" output or should we actually test the "esm" one ?